api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -1061,12 +1048,10 @@ public static void checkCoreDataset() {
 			log.error("Error while setting core roles for openmrs system", e);
 		}
 		finally {
-			Context.removeProxyPrivilege(PrivilegeConstants.MANAGE_ROLES);
```

Comment:
No point of having an empty finally block. Please remove that too..

Reply:
Am working on it.